tests/integration/CommandTesterTest.php

```diff
@@ -16,6 +16,9 @@ public function setUp()
 
     public function testInputApis()
     {
+        if (getenv('SCENARIO') == 'symfony4' && getenv('DEPENDENCIES') == 'lowest') {
+            $this->markTestSkipped('There is a bug with a lower dependency of symfony4 in how it handles tty.');
```

Comment:
Oh if this is only happening in LOWEST then we can just bump the minimum version of Symfony 4 in composer.json and we're all set. It's fine to just use what's in composer.lock for the symfony4 scenario as the lowest viable version. Ideally just bump the dependencies that need it.

`gtenv('SCENARIO')` isn't a reliable way to detect Symfony 4 anyway.